src/VehicleSetup/SetupView.qml

```diff
@@ -110,6 +111,13 @@ Rectangle {
                 panelLoader.sourceComponent = messagePanelComponent
             } else {
                 panelLoader.source = vehicleComponent.setupSource
+                for(var i = 0; i < componentRepeater.count; i++) {
+                    var obj = componentRepeater.itemAt(i);
+                    if (obj.text === vehicleComponent.name) {
+                        obj.checked = true;
```

Comment:
Marks the left button to be checked (so when you click on the summary
title, it selects the left  tab)
Em 20 de abr de 2016 22:43, "Don Gagne" <notifications@github.com> escreveu:

> In src/VehicleSetup/SetupView.qml
> <https://github.com/mavlink/qgroundcontrol/pull/3256#discussion_r60515590>
> :
>
> > @@ -110,6 +111,13 @@ Rectangle {
> >                  panelLoader.sourceComponent = messagePanelComponent
> >              } else {
> >                  panelLoader.source = vehicleComponent.setupSource
> > +                for(var i = 0; i < componentRepeater.count; i++) {
> > +                    var obj = componentRepeater.itemAt(i);
> > +                    if (obj.text === vehicleComponent.name) {
> > +                        obj.checked = true;
>
> What does this do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mavlink/qgroundcontrol/pull/3256/files/af054687eac80d5d2da7a23928f2e36965974358#r60515590>
>